purchase_order_block/models/purchase_order.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    po_block_id = fields.Many2one(
+        comodel_name='purchase.block.reason',
+        string='Purchase Block Reason')
+
+    released = fields.Boolean(copy=False, readonly=True)
```

Comment:
why use this field? It is released/unblocked if `po_block_id` is null.

Reply:
That was the requirement earlier. But now the requested changes are already made and have sent a request to the current Eficent branch